docs/tutorials/getting-started.md

```diff
@@ -57,7 +55,7 @@ $ juju set-model-constraints virt-type=virtual-machine mem=4G root-disk=8G
 For further details also consult the [Juju documentation on LXD](https://documentation.ubuntu.com/juju/3.6/reference/cloud/list-of-supported-clouds/the-lxd-cloud-and-juju/#the-lxd-cloud-and-juju)
 
 
-## Step 3: Deploy MicroCeph
+## Step 3: deploy MicroCeph
 
 With the Juju environment configured, the next step is to deploy MicroCeph. With the below will will deploy 3 clustered MicroCeph units:
```

Comment:
[* suggested rewording *] With the below commands... -> Deploy three-clustered MicroCeph units with this command: